app/lib/features/room/model/room_join_rule.dart

```diff
@@ -0,0 +1,2 @@
+// ignore: constant_identifier_names
+enum RoomJoinRule { Public, Private, Restricted }
```

Comment:
formerly, confusingly named `Visibility`. Notable: `spaceVisible` is now `Restricted` (in line with matrix spec naming)